server/src/main/java/org/opensearch/search/SearchHit.java

```diff
@@ -139,6 +142,8 @@ public final class SearchHit implements Writeable, ToXContentObject, Iterable<Do
 
     private Map<String, SearchHits> innerHits;
 
+    private FetchSearchResultProto.SearchHit searchHitProto;
```

Comment:
This should moved out to companion serde class, right?

Reply:
Right, I missed moving this.